Assets/Scripts/TileManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TileManager : MonoBehaviour
+{
+    
+    public GameObject tilePrefab;
+    public Material pathTileMaterial;
+    public Material nonPathTileMaterial;
+    public Tile[,] tiles;
+
+
+    //level 1 for right now. Stars represent the path
+    //and the - represent open sapces
+    private char[,] level1 = { 
+        {'*','-','-','-','-','*','*','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','*','*','*','*','*','-','*','-','-'},
+        {'-','-','-','-','-','-','-','*','-','-'},
+        {'-','*','*','*','*','*','*','*','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},};
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+        tiles = new Tile[10,10];
```

Comment:
Might want to move mapWidth and mapHeight to class vars instead of magic numbers so they're easier to change later if we need to.